B/BerkeleyDB/build_tarballs.jl

```diff
@@ -0,0 +1,65 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "BerkeleyDB"
+version = v"18.1.40"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://download.oracle.com/berkeley-db/db-$(version).tar.gz", "0cecb2ef0c67b166de93732769abdeba0555086d51de1090df325e18ee8da9c8")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+apk add gettext
+
```

Comment:
```suggestion
```

Reply:
Just a shot in the dark: I saw the message `autoreconf: configure.ac: not using Gettext`